recipe/meta.yaml

```
@@ -30,10 +30,10 @@ test:
   commands:
     - test -f ${PREFIX}/lib/libnettle${SHLIB_EXT}
     - test -f ${PREFIX}/lib/libhogweed${SHLIB_EXT}
-    - test -f ${PREFIX}/lib/libnettle.a
-    - test -f ${PREFIX}/lib/libhogweed.a
-  downstreams:
-    - gnutls
+#    - test -f ${PREFIX}/lib/libnettle.a
+#    - test -f ${PREFIX}/lib/libhogweed.a
```

```suggestion
    - test ! -f ${PREFIX}/lib/libnettle.a
    - test ! -f ${PREFIX}/lib/libhogweed.a
```
The suggested test asserts that these files do *not* exist.